podpac/core/data/interpolate.py

```diff
@@ -334,6 +335,8 @@ def can_interpolate(self, udims, source_coordinates, eval_coordinates):
 
         # confirm that udims are in both source and eval coordinates
         # TODO: handle stacked coordinates
+        # subselect unstacked coordinates for now
+        udims_subset = [d for d in udims_subset if d in source_coordinates.dims]
```

Comment:
This line allows multiple interpolators to be selected as part of the queue. Without it, only one interpolator that can handle all the dims will be selected.